### PR TITLE
V0.1.8 release parity

### LINKED ISSUE #311: v0.1.8 release parity
- [x] Datatype changes
- [x] Check for isSlashed in staker, stop execution if true
- [x] `UpdateCommission` has been introduced, instead of `setCommission` and `decreaseCommission`.

### PR DESCRIPTION
**Pull request by ashish10677**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #311 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Contracts version:
* Hardware:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by SkandaBhat — APPROVED** [review]
lgtm

File: cmd/claimBounty.go [hunk 1]
```diff
@@ -96,12 +96,12 @@ func claimBounty(config types.Configurations, client *ethclient.Client, redeemBo
 	}
 
 	log.Info("Claiming bounty transaction...")
-	waitFor := big.NewInt(1).Sub(bountyLock.RedeemAfter, big.NewInt(int64(epoch)))
-	if waitFor.Cmp(big.NewInt(0)) == 1 {
+	waitFor := bountyLock.RedeemAfter - epoch
+	if waitFor == 1 {
```
Comment (abhishekvispute) [inline cmd/claimBounty.go:100]: hmm its not given withdrawlockperiod is going to be 1 always 
we may change it, and with this we will have to change everywhere

same goes for other constants
so imo best approach would be having one common constants file

we can do this later, just pointing out

Comment (abhishekvispute) [reply cmd/claimBounty.go:100]: Also why this if is needed in first place ?

Comment (SkandaBhat) [reply cmd/claimBounty.go:100]: waitFor > 0

**Pull request merged**